source/styles.css

```diff
@@ -76,3 +76,26 @@
 
 .ReactVirtualized__List {
 }
+
+/* Tree default theme */
+.ReactVirtualized__Tree__node {
+  display: flex;
+  align-items: center;
+}
+
+.ReactVirtualized__Tree__node > :first-child {
+  margin-right: 10px;
+  width: 10px;
+}
+
+.ReactVirtualized__Tree__node button {
```

Comment:
The way RV treats styles is this:
* Functional styles (things that are important to the usability of a component) are defined as inline styles. This means they work even if the user doesn't want to import RV's default styles.
* Presentational styles are most avoid, with the exception of the `Table` component, in which case they're defined in this stylesheet.
* User can provide additional (external) CSS styles by specifying a `className` property.
* User can provide- even _override_ inline styles- by specifying a `style` property.

The `Tree` component styles defined in this CSS file are a mix of presentational and functional. Things like the border style are more presentational (and I slightly prefer we just stuck with the browser defaults for this sort of thing). But eg the width of the button and the margin are very much functional, because if the stylesheet is missing or removed, you end up with this:
<img width="105" alt="screen shot 2017-11-26 at 9 12 21 am" src="https://user-images.githubusercontent.com/29597/33242438-ea9a7fd8-d289-11e7-8255-e9818edd0966.png">

So I think the functional styles should be moved inline and the remaining presentational styles should either live in this file or just go away entirely and be up to the end user to specify.

Reply:
Good point. I'll fix it.